graspologic/match/gmp.py

```diff
@@ -159,7 +160,7 @@ def __init__(
             msg = '"padding" parameter must be of type string'
             raise TypeError(msg)
 
-    def fit(self, A, B, seeds_A=[], seeds_B=[]):
+    def fit(self, A, B, seeds_A=[], seeds_B=[], S=None, rng=None):
```

Comment:
random state is usually passed to constructor in sklearn as far as I know. what are your thoughts @asaadeldin11?

Reply:
sure, I'll change it

Reply:
should i also change it to `random_state` for consistency with sklearn @bdpedigo ?

Reply:
yes